zboxcore/sdk/sdk.go

```diff
@@ -1291,6 +1293,16 @@ func smartContractTxnValueFee(sn transaction.SmartContractTxnData,
 	txn.ToClientID = STORAGE_SCADDRESS
 	txn.Value = value
 	txn.TransactionFee = fee
+
+	// adjust fees if it's less
+	suggestedFee, err := txn.SuggestTransactionFeeFromMiners(blockchain.GetMiners(), 2)
```

Comment:
we don't need suggested fee is user provided it extensively

Reply:
what happens if it's 0 (or very minimal) or it's less than the suggested fee?

Reply:
We can use suggested fee only when the passed fee is 0, otherwise just use the fee user provided to avoid unnecessary requests.

Reply:
Makes sense. Done.